mod/state-transition/pkg/core/state_processor_staking.go

```diff
@@ -203,25 +208,31 @@ func (sp *StateProcessor[
 		return err
 	}
 	numWithdrawals := len(expectedWithdrawals)
+	if sp.cs.DepositEth1ChainID() == spec.BoonetEth1ChainID &&
+		slot >= spec.BoonetFork2Height {
+		// Enforce that there is at least one withdrawal for EVM inflation.
+		if numWithdrawals == 0 {
```

Comment:
nit: `stateWithdrawals` or `expectedWithdrawals` instead on `numWithdrawals`?

Reply:
`numWithdrawals` is derived from length of `expectedWithdrawals`.